demisto_sdk/tests/integration_tests/xsoar_linter_integration_test.py

```diff
@@ -9,9 +9,8 @@
 For a new checker, add the invalid statement in the relevant file and add it to the relevant test.
 """
 
-from pathlib import Path
-
 import pytest
+from wcmatch.pathlib import Path
```

Comment:
Why? please revert

Reply:
Lint expects the input to be `wcmatch.pathlib` and not `pathlib` (because it has a better `glob`).